strompris_api.py

```
@@ -49,7 +49,10 @@ def get_products(date):
         'Content-Type': 'application/json',
     }
 
-    response = requests.get(f'{BASE_URL}/feed/{date}', headers=headers)
+    expired_flag = 'true' if expired else 'false'
+
+    print(f'{BASE_URL}/feed/{date}?expired={expired_flag}')  # Include expired flag in the URL
+    response = requests.get(f'{BASE_URL}/feed/{date}?expired={expired_flag}', headers=headers)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)